draft-packages/well/tsconfig.dist.json

```diff
@@ -1,10 +1,5 @@
 {
   "extends": "../../tsconfig.dist.json",
-  "compilerOptions": {
-    "allowJs": false,
-    "declaration": true,
-    "noEmit": false,
-    "sourceMap": true
-  },
-  "exclude": ["docs"]
+  "include": ["index.ts", "*.tsx"],
```

Comment:
This one is different because the component files are put in the root and not within a folder like every other package 😬